web/webpack.config.js

```diff
@@ -43,7 +46,7 @@ module.exports = {
     ],
   },
   output: {
-    filename: '[name].js',
-    publicPath: '/',
+    path: path.resolve(__dirname),
```

Comment:
Maybe make this `path.resolve(__dirname, 'dist')` and that makes the gitignore easier.

Reply:
Yes, an output folder has my preference as well; it’s just really hard to get webpack to also move the html files into that output folder, and I would like html and js files to stick together. I made it work now; if we add more demo pages/files, I would consider trying out [Parcel](https://parceljs.org/) to see whether it better supports such an html-first instead of javascript-first approach.